.github/workflows/test.yml

```
@@ -82,10 +82,22 @@ jobs:
           path: |
             ./build/
 
+  runners-config:
+    name: Runners Configuration
+    runs-on: ubuntu-latest
+    outputs:
+      runners-config: ${{ steps.runners-config.outputs.runners-config }}
+    steps:
+      - name: Configure runners
+        run: |
```

actionlint (1.7.8)

property "stack_slug" is not defined in object type {}

(expression)